app/views/comments/edit.js.erb

```diff
@@ -0,0 +1 @@
+$("#<%= dom_id(@comment) %>").replaceWith("<%= j(render "comments/form", comment: @comment) %>");
```

Comment:
Good attention to detail using the short hand for escape_javascript() but not over-doing it for the arguments.